addons/website_project/controllers/main.py

```diff
@@ -51,6 +51,7 @@ def my_projects(self, page=1, date_begin=None, date_end=None, sortby=None, **kw)
 
         # content according to pager and archive selected
         projects = Project.search(domain, order=order, limit=self._items_per_page, offset=pager['offset'])
+        request.session['my_projects_history'] = projects.ids[:100]
```

Comment:
Thorically, It should be no more than 20 because website_portal set by default     _items_per_page = 20
and we just arbitrary take only max 100 to avoid to fill in the session with noise.

and no, it will no crash... because:

a = [1,2,3]
a[:200000] => [1,2,3]